CGPs/cgp-0042.md

```diff
@@ -0,0 +1,90 @@
+# CGP [0042]: cBRL contract deploy
```

Comment:
please follow the new template preamble here: https://github.com/celo-org/governance/blob/main/CGPs/cgp-template.md